Rock.Messaging/MQ/MessageCompression.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Text;
+﻿using RockLib.Immutable;
```

Comment:
This needs to be in a #if rocklib

Reply:
Good catch. I updated the PR with stuff that needed to be changed in order for the old solution to build.